### PR TITLE
Add LG Netcast remote doc

### PR DESCRIPTION
**Pull request by doudz**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#19889

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/remote.lg_netcast.markdown [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+---
+
+The `lg_netcast` remote platform allows you to control the state of your LG Netcast TV, NetCast 3.0 (LG Smart TV models released in 2012) and NetCast 4.0 (LG Smart TV models released in 2013).
+
+
+```yaml
+# Example configuration.yaml entry
+remote:
+  - platform: lg_netcast
+    name: tv_room
```
Comment (frenck) [inline source/_components/remote.lg_netcast.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

File: source/_components/remote.lg_netcast.markdown [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    host: 192.168.0.20
+```
+
+{% configuration %}
+host:
+  description: The IP address of the LG Smart TV, e.g., 192.168.0.20.
```
Comment (frenck) [inline source/_components/remote.lg_netcast.markdown]:
```suggestion
  description: The IP address of the LG Smart TV, e.g., `192.168.0.20`.
```

File: source/_components/remote.lg_netcast.markdown [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+  required: false
+  type: string
+name:
+  description: The name you would like to give to the LG Smart TV.
+  required: false
+  default: LG TV Remote
+  type: string
+{% endconfiguration %}
+
+To get the access token for your TV configure the `lg_netcast` platform in Home Assistant without the `access_token`.
```
Comment (frenck) [inline source/_components/remote.lg_netcast.markdown]:
```suggestion
To get the access token for your TV, configure the `lg_netcast` platform in Home Assistant without the `access_token`.
```

File: source/_components/remote.lg_netcast.markdown [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+After starting Home Assistant the TV will display the access token on screen.
```
Comment (frenck) [inline source/_components/remote.lg_netcast.markdown]:
```suggestion
After starting Home Assistant, the TV will display the access token on screen.
```

File: source/_components/remote.lg_netcast.markdown [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+Just add the token to your configuration and restart Home Assistant and the media player component for your LG TV will show up.
+
+<p class='note'>
+The access token will not change until you factory reset your TV.
+</p>
+
+### {% linkable_title Service `remote.turn_off` %}
+
+Turn off device.
```
Comment (frenck) [inline source/_components/remote.lg_netcast.markdown]:
```suggestion
Turn off the device.
```

File: source/_components/remote.lg_netcast.markdown [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+    AV_MODE = 410
+    SIMPLINK = 411
+    EXIT = 412
+    RESERVATION_PROGRAM_LIST = 413
+    PIP_CHANNEL_UP = 414
+    PIP_CHANNEL_DOWN = 415
+    SWITCH_VIDEO = 416
+    APPS = 417
+```
+
```
Comment (frenck) [inline source/_components/remote.lg_netcast.markdown:157]: Extra unneeded empty line.
```suggestion
```

File: source/_components/remote.lg_netcast.markdown [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+description: "Instructions on how to integrate LG Netcast (Netcast 3.0 & 4.0) remotes into Home Assistant."
+date: 2016-11-05 17:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: lg.png
+ha_category: Remote
+ha_iot_class: "Local Push"
+ha_release: "0.82.0"
```
Comment (frenck) [inline source/_components/remote.lg_netcast.markdown]:
```suggestion
ha_release: 0.85
```

File: source/_components/remote.lg_netcast.markdown [hunk 8]
```diff
@@ -0,0 +1,154 @@
+---
+layout: page
+title: "LG Netcast Remote"
+description: "Instructions on how to integrate LG Netcast (Netcast 3.0 & 4.0) remotes into Home Assistant."
+date: 2016-11-05 17:00
```
Comment (frenck) [inline source/_components/remote.lg_netcast.markdown]: Please update this date to the creation date.

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks, @doudz! I've reviewed your PR and left some comments. Could you please take a look? 👍

**Comment by doudz** [comment]
I think everything's ok now ?

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by frenck** [comment]
Closing this PR, since the parent PR has been closed.

**Pull request closed without merging**